helpers/stable_swap_math/mod.rs

```diff
@@ -66,19 +69,21 @@ fn compute_d(amounts: &Vec<u128>, amp_coef: u128) -> Result<U256, MathError> {
         // Computes next D unitl satisfying precision is reached
         for _ in 0..MAX_ITERATIONS {
             let d_next = compute_d_next(d, n, amounts, ann_sum, ann_sub_one, n_add_one)?;
-            if d_next > d {
-                if d_next.checked_sub(d).ok_or(MathError::SubUnderflow(2))? <= 1.into() {
-                    return Ok(d);
-                }
-            } else if d.checked_sub(d_next).ok_or(MathError::SubUnderflow(3))? <= 1.into() {
+            // [nit] Maybe like this?
+            if d_next.abs_diff(d) <= 1.into() {
```

Comment:
I was looking for this the first time I reviewed the code and then forget to comment about it <3